airflow/contrib/hooks/athena_hook.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from builtins import str
+import logging
+
+import pyathenajdbc
+
+from airflow.hooks.dbapi_hook import DbApiHook
+
+
+class AthenaException(Exception):
```

Comment:
With Python you don't need to-do this. See: https://stackoverflow.com/questions/1319615/proper-way-to-declare-custom-exceptions-in-modern-python

The exception gets bubbled up, but it's just called an AthenaException instead of a generic "value errors" etc.